client/state/products-list/selectors.js

```diff
@@ -33,3 +43,40 @@ export function getProductDisplayCost( state, productSlug ) {
 
 	return product.cost_display;
 }
+
+export const getPlanPrice = ( state, siteId, plan, isMonthlyPreference ) => {
```

Comment:
I won't block on this, but a general note that we've moved to global selectors in `client/state/selector` mainly for ease of discovery and importing. 

Main difference is one selector per file, and naming such that folks are able to tell what we're retrieving without additional context.

Please keep this in mind for next time. 👍

Reply:
Do you mind adding JSDoc for the new selectors? For example explaining when `isMonthlyPreference` is used would help. 

Reading the code this would mean break this into price per month unless the plan is already monthly?

Non-blocking again, but I think the interface may be made a bit clearer, and maybe we could follow up in another PR when we see other usage.

For example, we might consider something like:

`getPlanPricePerMonth( state, stateId, plan )`

or something more generic, where term might be monthly, yearly, biyearly etc.

`getPlanPricePerTerm( state, stateId, plan, interval)`